src/main/java/org/cryptomator/common/locationpresets/DropboxMacLocationPresetsProvider.java

```diff
@@ -0,0 +1,28 @@
+package org.cryptomator.common.locationpresets;
+
+import org.cryptomator.integrations.common.CheckAvailability;
+import org.cryptomator.integrations.common.OperatingSystem;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Stream;
+
+import static org.cryptomator.integrations.common.OperatingSystem.Value.MAC;
+
+@OperatingSystem(MAC)
+@CheckAvailability
+public final class DropboxMacLocationPresetsProvider implements LocationPresetsProvider {
+
+	private static final Path LOCATION = LocationPresetsProvider.resolveLocation("~/Library/CloudStorage/Dropbox");
```

Comment:
Does not exist on my machine, despite Dropbox being available. What about `~/Dropbox`?

Reply:
Fixed in https://github.com/cryptomator/cryptomator/pull/2918/commits/1d6edb8373d9598d80cdf21e1de80c0fd1956b80